Camera/Basic/CaptureHDRCompleteSettings/CaptureHDRCompleteSettings.cpp

```diff
@@ -0,0 +1,73 @@
+/*
+This example shows how to acquire an HDR image from the Zivid camera with fully
+configured settings for each frame. In general, taking an HDR image is a lot
+simpler than this as the default settings work for most scenes. The purpose of
+this example is to demonstrate how to configure all the settings.
+*/
+
+#include <Zivid/CloudVisualizer.h>
+#include <Zivid/Zivid.h>
+
+#include <iostream>
+
+int main()
+{
+    try
+    {
+        Zivid::Application zivid;
+
+        std::cout << "Setting up visualization" << std::endl;
+        Zivid::CloudVisualizer vis;
+        zivid.setDefaultComputeDevice(vis.computeDevice());
+
+        std::cout << "Connecting to the camera" << std::endl;
+        auto camera = zivid.connectCamera();
+
+        std::cout << "Configuring settings same for all HDR frames" << std::endl;
+        auto settingsDefault = Zivid::Settings();
+        settingsDefault.set(Zivid::Settings::Brightness{ 1 })
+            .set(Zivid::Settings::Gain{ 1 })
+            .set(Zivid::Settings::Bidirectional{ false })
+            .set(Zivid::Settings::Filters::Contrast::Enabled::yes)
+            .set(Zivid::Settings::Filters::Contrast::Threshold{ 5 })
+            .set(Zivid::Settings::Filters::Gaussian::Enabled::yes)
+            .set(Zivid::Settings::Filters::Gaussian::Sigma{ 1.5 })
+            .set(Zivid::Settings::Filters::Outlier::Enabled::yes)
+            .set(Zivid::Settings::Filters::Outlier::Threshold{ 5 })
+            .set(Zivid::Settings::Filters::Reflection::Enabled::yes)
+            .set(Zivid::Settings::Filters::Saturated::Enabled::yes)
+            .set(Zivid::Settings::BlueBalance{ 1.081 })
+            .set(Zivid::Settings::RedBalance{ 1.709 });
+
+        std::cout << "Configuring settings different for all HDR frames" << std::endl;
```

Comment:
either "differently", or "Configuring different settings for all HDR frames". But, shouldn't it also say ".. for all frames in the HDR merge". Just to clarify that it's one HDR that consists of multiple frame, and not multiple HDRs... It should be clear... but it might not be.

Reply:
I agree it should be clear, not easy though. This is my current proposal:

Configuring settings same for all frames in the HDR frame
Configuring settings differently for all frames in the HDR frame

I wanted to get rid of 'merge'. Both single captures and HDRs are frames so I think above makes sense?

Reply:
Configuring settings differently for all frames in the HDR frame

Reply:
done